source/connection/network-compression.txt

```diff
@@ -1,95 +1,101 @@
-You can enable a driver option to compress messages which reduces the amount
-of data passed over the network between MongoDB and your application.
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
 
-The driver supports the following algorithms:
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: zlib, zstandard, zstd, snappy
 
-1. `Snappy <https://google.github.io/snappy/>`__: available in MongoDB 3.4 and later.
+Overview
+--------
 
-2. `Zlib <https://zlib.net/>`__: available in MongoDB 3.6 and later.
+In this guide, you can learn how to use the {+driver-short+} to enable network
+compression. The driver provides a connection option to compress messages, which
+reduces the amount of data passed over the network between MongoDB and your application.
 
-3. `Zstandard <https://github.com/facebook/zstd/>`__: available in MongoDB 4.2 and later.
+The driver supports the following compression algorithms:
+
+- `Snappy <https://google.github.io/snappy/>`__: Available in {+mdb-server+} v3.4 and later.
+- `Zlib <https://zlib.net/>`__: Available in {+mdb-server+} v3.6 and later.
+- `Zstandard <https://github.com/facebook/zstd/>`__: Available in {+mdb-server+} v4.2 and later.
 
 The driver tests against the following versions of these libraries:
 
 - ``{+snappyVersion+}``
 - ``{+zstdVersion+}``
 
-If you specify multiple compression algorithms, the driver selects the
-first one in the list supported by the MongoDB instance to which it is
-connected.
+If you specify multiple compression algorithms, the driver selects the first one
+in the list supported by your MongoDB instance.
 
 .. note::
 
    Applications that require Snappy or Zstandard compression must
-   :ref:`add explicit dependencies <compression-dependencies>` for those
-   algorithms.
+   add explicit dependencies for those algorithms. To learn more,
+   see the :ref:`java-compression-dependencies` section of this guide.
 
-.. _enable-compression:
+.. _java-compression-specify:
 
 Specify Compression Algorithms
 ------------------------------
 
 You can enable compression for the connection to your MongoDB instance
-by specifying the algorithms in one of two ways: adding the parameter to your
-connection string using ``ConnectionString`` or by calling the method in the
-``MongoClientSettings.Builder`` class.
-
-.. tabs::
+by specifying the algorithms in one of the following ways: 
 
-   .. tab:: ConnectionString
-      :tabid: connectionstring
-
-      To enable compression using the `ConnectionString <{+api+}/apidocs/mongodb-driver-core/com/mongodb/ConnectionString.html>`__,
-      add the parameter ``compressors`` in the connection string passed to
-      ``MongoClients.create()``. You can specify one or more compression
-      algorithms, separating them with commas:
+- Use the ``compressors`` parameter in your connection string
+- Chain the ``compressorList()`` method to the ``MongoClientSettings.builder()`` method
```

Comment:
```suggestion
- Use the ``compressors`` parameter in your connection string.
- Chain the ``compressorList()`` method to the ``MongoClientSettings.builder()`` method.
```
S: Would use a period since there is an implied you.